app/views/users/show.html.erb

```diff
@@ -1 +1,26 @@
-<h1 name='title'>Here is most of the information for User ID: <%= @user_id %></h1>
\ No newline at end of file
+<head>
+  <%= stylesheet_link_tag "user_show" %>
+</head>
+<body class="mrg">
+  <div>
+    <%= render partial: 'user', locals: { user: @user } %>
+  </div>
+  <div class="border mrg">
+    <div class="title mrg">
+      <h2>Bio</h2>
+      <p><%= @user.bio %></p>
+    </div>
+  </div>
+  <div>
+    <% if @posts.present? %>
+      <% @posts.each_with_index do |post, index| %>
+        <%= render partial: 'posts/post', locals: { post: post, index: index + 1 } %>
+      <% end %>
+    <% else %>
+      <p>No posts available</p>
+    <% end %>
+  </div>
+  <div class="btn">
+    <button type="button">See all posts!</button>
```

Comment:
- [ ] This button should open a user's post index as displayed in the wireframe and project requirements. Currently, it leads to nowhere